src/bandersnatch/utils.py

```diff
@@ -118,6 +116,15 @@ def recursive_find_files(files: Set[Path], base_dir: Path) -> None:
         recursive_find_files(files, directory)
 
 
+def find_all_files(files: Set[Path], base_dir: Path) -> None:
+    for f in base_dir.rglob("*"):
+        if not f.is_file():
+            continue
+        if hasattr(f, "keep_file") and f.name == f.keep_file:  # type: ignore
+            continue
+        files.add(f)
```

Comment:
Thanks for the move. Lets ensure we get a unittest around here - I'm a big fan of util functions always having good unittest coverage as they should be able to (and this can be) easily tested :)